example_configs/flink.yml

```diff
@@ -0,0 +1,58 @@
+rules:
+  ## Job Manager ##
+  # Example: org.apache.flink.metrics<key0=127.0.0.1, key1=jobmanager, name=taskSlotsTotal><>Value
+  - pattern: org.apache.flink.metrics<key0=(.*), key1=jobmanager, name=([a-z].*)><>Value
+    name: flink_jobmanager_$2
+    labels:
+      ip: $1
+
+  # Example: org.apache.flink.metrics<key0=127.0.0.1, key1=jobmanager, key2=My_Job, name=lastCheckpointDuration><>Value
+  - pattern: org.apache.flink.metrics<key0=(.*), key1=jobmanager, key2=(.*), name=([a-z].*)><>Value
+    name: flink_jobmanager_$3
+    labels:
+      ip: $1
+      job: $2
+
+  ## Task Manager ##
+  # Example: org.apache.flink.metrics<key0=debian-8, key1=taskmanager, key2=407401ecfef1f050d3dcc83af9cbfb74, key3=My_Job, key4=Map, key6=1, name=numRecordsIn><>Count
+  - pattern: org.apache.flink.metrics<key0=(.*), key1=taskmanager, key2=(.*), key3=(.*), key4=(.*), key5=([0-9]*), name=([a-z].*)><>Count
+    name: flink_taskmanager_$6
+    labels:
+      host: $1
```

Comment:
key0 in taskmanager metrics is set to the hostname on which taskmanager is running.
So, on my dev environment it's set to `debian-8` (which is also the output of `hostname`)
On one of the prod nodes, it's set to the `hostname`.